src/backend/weaviate/weaviate_manager.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import weaviate
+from weaviate import Client
+
+#config logging
+logging.basicConfig(level=logging.DEBUG,format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(__name__)
+
+class WeaviateManager:
+    def __init__(self):
+
+        try:
+            self.client=weaviate.Client("http://weaviate:8080")
+            logger.info("Successfully connected to Weaviate")
+        except Exception as e:
+            logger.error(f"Failed to initialize Weaviate connection: {e}")
+            raise e
+
+    def add_weaviate_customer_class(self,customer_guid):
+    #Add a new customer class in Weaviate using customer_guid
+        valid_class_name=f"Customer_{customer_guid.replace('-','_')}"
```

Comment:
Can we enable this class name to be returned by a function so that it can be reused